sdk/eventhub/Azure.Messaging.EventHubs.Processor/src/EventProcessorClient.cs

```diff
@@ -1050,16 +1076,9 @@ private async Task StartPartitionProcessingAsync(string partitionId)
                 }
             }
 
-            // TODO: if reason = Shutdown or OwnershipLost and we got an exception when closing, what should the final reason be?
-
-            if (!PartitionContexts.TryRemove(partitionId, out var context))
-            {
-                context = new PartitionContext(partitionId);
-            }
```

Comment:
We can delete the `PartitionContexts` dictionary. We only used it for this reason.

Reply:
Missed that.  Thanks!